exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/JsonConvertFrom.java

```diff
@@ -76,7 +77,7 @@ public void eval(){
     @Output ComplexWriter writer;
 
     public void setup(){
-      jsonReader = new org.apache.drill.exec.vector.complex.fn.JsonReader(buffer, false, false, false /* do not read numbers as doubles */);
+      jsonReader = new org.apache.drill.exec.vector.complex.fn.JsonReader(buffer, false, false, false, false /* do not read numbers as doubles */);
```

Comment:
See above. Why do we have two copies? Can we have a function that returns the reader using default configs?